code/modules/projectiles/projectile/magic.dm

```diff
@@ -734,3 +736,22 @@
 	armor_penetration = 100
 	temperature = 50
 	flag = "magic"
+
+/obj/projectile/magic/water
+	name = "bolt of water"
+	icon_state = "ice_2"
+	flag = "magic"
+
+/obj/projectile/magic/water/on_hit(target)
+	. = ..()
+	var/obj/item/reagent_containers/K = new /obj/item/reagent_containers/glass/bucket/wooden/SW(get_turf(target))
```

Comment:
Does this reagent container ever have reagents added to it? You later call K.reagents.reaction(mob_target, TOUCH), hence I assume there would need to be some?

Reply:
It has reagents added to it on creation yes, within the Seelie spells file because its a special bucket specifically for the spell.
![image](https://github.com/user-attachments/assets/0f7bd14e-79fd-4c41-8a98-d68da9d4a27e)

Reply:
Cursed typepath, I suppose this works- if you feel like it you could change it to be more clear but I'll leave that up to you!